ssl/ssl_locl.h

```diff
@@ -1854,6 +1854,8 @@ const char *ssl_protocol_to_string(int version);
 
 # ifndef OPENSSL_UNIT_TEST
 
+__owur int ssl_read_internal(SSL *s, void *buf, size_t num, size_t *readbytes);
+__owur int ssl_write_internal(SSL *s, const void *buf, size_t num, size_t *written);
```

Comment:
Ah, now I see. I must have scrolled past that difficult a dozen times without noticing. Ok then...